applications/RomApplication/python_scripts/empirical_cubature_method.py

```diff
@@ -17,7 +17,7 @@ class EmpiricalCubatureMethod():
     def __init__(
         self,
         ECM_tolerance = 0,
-        Filter_tolerance = 1e-16,
+        Filter_tolerance = 0,
```

Comment:
Why is the tolerance here set to 0 by default? 
I mean, do you think it will most likely need to use all elements/conditions to converge?